crates/aptos/src/governance/mod.rs

```diff
@@ -630,14 +634,13 @@ impl SubmitVote {
             let remaining_voting_power = self
                 .args
                 .txn_options
-                .view(ViewRequest {
-                    function: "0x1::aptos_governance::get_remaining_voting_power"
-                        .parse()
-                        .unwrap(),
-                    type_arguments: vec![],
-                    arguments: vec![
-                        serde_json::Value::String(pool_address.to_string()),
-                        serde_json::Value::String(proposal_id.to_string()),
+                .view(ViewFunction {
+                    module: ModuleId::new(AccountAddress::ONE, ident_str!("voting").to_owned()),
```

Comment:
This should be `aptos_governance` right?

Reply:
Fixed